apps/src/lib/levelbuilder/lesson-editor/ActivityCard.jsx

```diff
@@ -0,0 +1,229 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import color from '@cdo/apps/util/color';
+import {borderRadius} from '@cdo/apps/lib/levelbuilder/constants';
+import OrderControls from '@cdo/apps/lib/levelbuilder/OrderControls';
+import ActivitySectionCard from '@cdo/apps/lib/levelbuilder/lesson-editor/ActivitySectionCard';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {
+  addActivitySection,
+  moveActivity,
+  removeActivity,
+  updateActivityField
+} from '@cdo/apps/lib/levelbuilder/lesson-editor/activitiesEditorRedux';
+import ReactDOM from 'react-dom';
+import {activityShape} from '@cdo/apps/lib/levelbuilder/shapes';
+
+const styles = {
+  activityHeader: {
+    fontSize: 18,
+    color: 'white',
+    background: color.cyan,
+    borderTopLeftRadius: borderRadius,
+    borderTopRightRadius: borderRadius,
+    padding: 10,
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center'
+  },
+  activityBody: {
+    background: color.lightest_cyan,
+    borderBottomLeftRadius: borderRadius,
+    borderBottomRightRadius: borderRadius,
+    padding: 10,
+    marginBottom: 20
+  },
+  addButton: {
+    fontSize: 14,
+    color: '#5b6770',
+    background: 'white',
+    border: '1px solid #ccc',
+    boxShadow: 'none',
+    margin: '0 10px 10px 10px'
+  },
+  button: {
+    marginLeft: 10
+  },
+  label: {
+    fontSize: 20,
+    marginRight: 5
+  },
+  input: {
+    marginRight: 10
+  },
+  labelAndInput: {
+    marginLeft: 5,
+    display: 'flex',
+    alignItems: 'center'
+  }
+};
+
+class ActivityCard extends Component {
```

Comment:
please add a comment explaining that this contains the edit controls for an Activity